erdpy/proxy/cost.py

```diff
@@ -0,0 +1,80 @@
+import base64
+
+from erdpy.proxy.tx_types import TxTypes
+from erdpy import utils, errors
+from erdpy.projects import load_project
+
+
+def prepare_argument(argument):
```

Comment:
This function is also present in `nodedebug.py`. It is a bit of duplication, but leave it as it is now, because we will soon add a `transaction.py` module and move there some behavior and some logic.

Reply:
ok